### PR TITLE
Update FAQ to address common questions we are seeing

### PR DESCRIPTION
**Pull request by k-nearest-neighbor**
Here is a doc to collaborate on these 

https://docs.google.com/document/d/1XZJcL_LxZfYUPZnihvckn7I1iEQaV0qBdobMQ6W2O04/edit?usp=sharing

### CODE REVIEW

File: website/public/locales/en/index.json [hunk 1]
```diff
@@ -6,7 +6,15 @@
...
-    "a1": "Open Assistant is a project organized by LAION and individuals around the world interested in bringing this technology to everyone."
+    "a1": "Open Assistant is a project organized by LAION and individuals around the world interested in bringing this technology to everyone.",
+    "q2": "What license does Open Assistant use?",
+    "a2": "The code and models are licensed under the Apache 2.0 license.",
+    "q3": "Will the training data also be released?",
+    "a3": "Yes, also under Apache 2.0.",
+    "q4": "Will Open Assistant be free?",
+    "a4": "Yes, Open Assistant will be free to use and modify.",
+    "q5": "What hardware will be required to run the models?",
+    "a5": "There will be versions which will be runnable on consumer hardware. 8GB of VRAM."
```
Comment (AbdBarho) [inline website/public/locales/en/index.json:17]: I would not say 8GB of ram, just consumer hardware. we don't know any of this yet

Comment (k-nearest-neighbor) [reply website/public/locales/en/index.json:17]: 👍

**Comment by k-nearest-neighbor** [comment]
Trying to understand the pre-commit error about poetry configuration. Pre-commit passed locally.

**Comment by AbdBarho** [comment]
> Trying to understand the pre-commit error about poetry configuration. Pre-commit passed locally.

Very weird error, especially since you didn't touch the python code, probably a problem with some dependency.

**Review by AbdBarho — APPROVED** [review]

File: website/public/locales/en/index.json [hunk 2]
```diff
@@ -6,7 +6,15 @@
...
-    "a1": "Open Assistant is a project organized by LAION and individuals around the world interested in bringing this technology to everyone."
+    "a1": "Open Assistant is a project organized by LAION and individuals around the world interested in bringing this technology to everyone.",
+    "q2": "What license does Open Assistant use?",
+    "a2": "The code and models are licensed under the Apache 2.0 license.",
+    "q3": "Will the training data also be released?",
+    "a3": "Yes, under CC BY 4.0.",
+    "q4": "Will Open Assistant be free?",
+    "a4": "Yes, Open Assistant will be free to use and modify.",
+    "q5": "What hardware will be required to run the models?",
+    "a5": "There will be versions which will be runnable on consumer hardware."
```
Comment (yk) [inline website/public/locales/en/index.json:17]: I'd say something like "we try our best to make X happen" :)

**Review by yk — APPROVED** [review]

**Pull request merged**